ext/mri/2.3.0/id_table.h

```diff
@@ -0,0 +1,30 @@
+#ifndef RUBY_ID_TABLE_H
+#define RUBY_ID_TABLE_H 1
+#include "ruby/ruby.h"
+
+struct rb_id_table;
```

Comment:
I tried casting using this struct instead, but it really expect a `st_data_t`.

```
compiling mri/mri.c
mri/mri.c: In function ‘internal_instance_methods’:
mri/mri.c:104:51: error: dereferencing pointer to incomplete type
mri/mri.c:104:89: warning: passing argument 3 of ‘st_foreach’ makes integer from pointer without a cast [enabled by default]
/home/travis/.rvm/rubies/ruby-2.3.1/include/ruby-2.3.0/ruby/st.h:122:5: note: expected ‘st_data_t’ but argument is of type ‘struct rb_id_table *’
mri/mri.c: In function ‘Looksee_internal_undefined_instance_methods’:
mri/mri.c:142:51: error: dereferencing pointer to incomplete type
mri/mri.c:142:90: warning: passing argument 3 of ‘st_foreach’ makes integer from pointer without a cast [enabled by default]
/home/travis/.rvm/rubies/ruby-2.3.1/include/ruby-2.3.0/ruby/st.h:122:5: note: expected ‘st_data_t’ but argument is of type ‘struct rb_id_table *’
make: *** [mri.o] Error 1
```

https://travis-ci.org/mathieujobin/looksee/jobs/138376615

if a typedef is missing in this file, would that be a bug in ruby 2.3 ? 

I don't know what this empty struct does? I don't remember seeing struct with no members.

Thanks for taking a look at this, much appreciated.